packages/dom-adapters/src/caret/utils/absoluteOffset.ts

```diff
@@ -0,0 +1,57 @@
+/**
+ * Returns true if node is a line break
+ *
+ * <div>
+ *   <div>
+ *     <br>
+ *   </div>
+ * </div>
+ *
+ * @param node - node to check
+ */
+function isLineBreak(node: Node): boolean {
```

Comment:
Could you move it below the exported members

Reply:
There will be ES Lint error: using of method before it declared